src/sentry/notifications/helpers.py

```diff
@@ -263,6 +263,7 @@ def get_scope_type(type: NotificationSettingTypes) -> NotificationScopeType:
         NotificationSettingTypes.QUOTA_REPLAYS,
         NotificationSettingTypes.QUOTA_WARNINGS,
         NotificationSettingTypes.QUOTA_SPEND_ALLOCATIONS,
+        NotificationSettingTypes.MISSING_MEMBERS,
```

Comment:
@cathteng as mentioned in DM, we can probably leverage the existing approval notification type since it's used for other nudges

Reply:
done